src/models/model.cpp

```diff
@@ -37,9 +37,9 @@ static std::string CurrentModulePath() {
 
 namespace Generators {
 
-State::State(const GeneratorParams& params, const Model& model)
+State::State(const GeneratorParams& params_, const Model& model)
```

Comment:
Was this rename accidental?

Reply:
I had some compilation issues with the original code, let me retest it, thanks.